qmxgraph/widget.py

```diff
@@ -396,10 +396,15 @@ def _on_drop(self, event):
 
             # Refer to `mime.py` for docs about format
             version = parsed['version']
-            if version == 1:
-                x = event.pos().x()
-                y = event.pos().y()
+            if version not in (1, 2):
```

Comment:
As we discussed, please review if this versioning of the mime data is necessary, and if it is please document it. 👍